site/en/install/source_windows.md

```diff
@@ -131,7 +131,7 @@ Please specify the cuDNN version you want to use. [Leave empty to default to cuD
 Please specify the location where cuDNN 7 library is installed. Refer to README.md for more details. [Default is C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.0]: <b>C:\tools\cuda</b>
 
 Please specify a list of comma-separated Cuda compute capabilities you want to build with.
-You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
+You can find compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
```

Comment:
Please review software-suggested changes and take into account things like context. Reversing this change:

```suggestion
You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
```